### PR TITLE
fix Routes::Route::route deprecation for Mojolicious 8.67+

### PR DESCRIPTION
**Pull request by dmanto**
Starting on Mojolicious 8.67, Mojolicious::Routes::Route::route is deprecated in favor of Mojolicious::Routes::Route::any. This PR is to use new recommended way (any instead of route, methods instead of via).

Also adds .gitattributes file to help github detect Perl language (instead of marking files as Raku's)

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*.pl linguist-language=Perl
+*.pm linguist-language=Perl
+*.t linguist-language=Perl
```
Comment (jhthorsen) [inline .gitattributes:3]: Please make a separate PR for this.

File: .github/workflows/ci-ubuntu.yml [hunk 2]
```diff
@@ -6,12 +6,17 @@ jobs:
     strategy:
       matrix:
         perl-version:
-          - '5.30'
+          - "5.30"
```
Comment (jhthorsen) [inline .github/workflows/ci-ubuntu.yml:9]: No need to change this.

File: .github/workflows/ci-ubuntu.yml [hunk 3]
```diff
@@ -6,12 +6,17 @@ jobs:
...
       image: perl:${{matrix.perl-version}}
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2-beta
+      - name: install node 14.x on perl's container
+        run: |
+          apt-get update
+          apt-get install -y curl
+          curl -sL https://deb.nodesource.com/setup_14.x | bash -
+          apt-get install -y nodejs
```
Comment (jhthorsen) [inline .github/workflows/ci-ubuntu.yml:19]: I don't understand which value this adds. It also seems like this is not related to the this PR.

File: lib/Mojolicious/Plugin/Webpack.pm [hunk 4]
```diff
@@ -18,7 +18,7 @@ sub route      { shift->{route} }
 
 sub register {
   my ($self, $app, $config) = @_;
-  $self->{route} = $app->routes->route('/asset/*name')->via(qw(HEAD GET))->name('webpack.asset');
+  $self->{route} = $app->routes->any('/asset/*name')->methods(qw(HEAD GET))->name('webpack.asset');
```
Comment (jhthorsen) [inline lib/Mojolicious/Plugin/Webpack.pm:21]: This will break old versions of Mojolicious. Please have a look at https://github.com/jhthorsen/mojolicious-plugin-assetpack/pull/142/files to see how it can be done without breaking compatibility.

File: lib/Mojolicious/Plugin/Webpack/Builder.pm [hunk 5]
```diff
@@ -14,9 +14,9 @@ our $VERSION = $Mojolicious::Plugin::Webpack::VERSION || '0.01';
 
 has dependencies => sub {
   return {
-    core => [qw(webpack webpack-cli webpack-plugin-hash-output html-webpack-plugin)],
-    css  => [qw(css-loader mini-css-extract-plugin optimize-css-assets-webpack-plugin)],
-    js   => [qw(@babel/core @babel/preset-env babel-loader terser-webpack-plugin)],
+    core => [qw(webpack webpack-cli html-webpack-plugin@next)],
+    css  => [qw(css-loader mini-css-extract-plugin css-minimizer-webpack-plugin)],
+    js   => [qw(@babel/core @babel/preset-env babel-loader)],
```
Comment (jhthorsen) [inline lib/Mojolicious/Plugin/Webpack/Builder.pm:19]: I don't understand these changes either.

File: t/public/asset/webpack.development.html [hunk 6]
```diff
@@ -1,8 +1,60 @@
-<!DOCTYPE html>
```
Comment (jhthorsen) [inline t/public/asset/webpack.development.html:1]: Is this file supposed to be part of the PR? It looks very bad.

**Review by jhthorsen — CHANGES_REQUESTED** [review]
Looks like there's a lot of unrelated changes that makes me wonder what the PR is all about.

**Comment by jhthorsen** [comment]
The actual issue mentioned in this PR is now fixed in master. Please make smaller PRs for the other changes.

File: .github/workflows/ci-ubuntu.yml (hunk 3 above)
Comment (dmanto) [reply .github/workflows/ci-ubuntu.yml:19]: that would install a recent nodejs version on perl's container.
CI script will fail without that

File: .github/workflows/ci-ubuntu.yml (hunk 2 above)
Comment (dmanto) [reply .github/workflows/ci-ubuntu.yml:9]: Sorry, didn't mean that. Have to check vscode reformatter for yml files

File: lib/Mojolicious/Plugin/Webpack.pm (hunk 4 above)
Comment (dmanto) [reply lib/Mojolicious/Plugin/Webpack.pm:21]: I see you fix  this on master branch, thanks

File: lib/Mojolicious/Plugin/Webpack/Builder.pm (hunk 5 above)
Comment (dmanto) [reply lib/Mojolicious/Plugin/Webpack/Builder.pm:19]: These changes are meant to support version 5 of webpack, which has a lot of breaking changes. I added those because without them tests wouldn't pass when npm installs that new (default) version.
They are big changes though, and I am not sure of the full implicances. I have a version running in my computer with these changes and running webpack 5, and seem to work fine, but other than that I can't tell.
I think the main issue here is to decide which webpack version will be the main supported version, and if the plugin should support older / newer versions.
